spring-cloud-deployer-local/src/test/java/org/springframework/cloud/deployer/spi/local/LocalAppDeployerEnvironmentIntegrationTests.java

```diff
@@ -242,6 +246,46 @@ public void testInDebugModeWithSuspended() throws Exception {
 		deployer.undeploy(deploymentId);
 	}
 
+	@Test
+	public void testActuatorOperations() {
+		if (useDocker) {
```

Comment:
I think we can use the Spring version of [@DisabledIf](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/test/context/junit/jupiter/DisabledIf.html) to guard the method on the property such as
```java
 @DisabledIf("${spring-cloud-deployer-spi-test-use-docker}")
```

Not sure how many other places we are doing the docker short-circuit though.

Reply:
@onobc There has always been a "certification" test suite for all AppDeployer implementations must subclass.  It appears that, at some point, somebody realized that deploying a docker container would be a good test, and the LocalAppDeployer gets a pass because it only does bare metal JVM deployment.

Reply:
Good idea anyway :)